Client/game_sa/CFxManagerSA.cpp

```diff
@@ -67,6 +67,32 @@ void CFxManagerSA::OnFxSystemSAInterfaceDestroyed(CFxSystemSAInterface* pFxSyste
         delete pFxSystemSA;
 }
 
+CFxSystemBPSAInterface* CFxManagerSA::GetFxSystemBlueprintByName(SString sName)
+{
+    DWORD dwThis = (DWORD)m_pInterface;
+    DWORD dwFunc = FUNC_FxManager_c__GetSystemByName;
+    const char* pChars = sName;
+    CFxSystemBPSAInterface* pFxSystemBlueprint = nullptr;
+
+    _asm

```

Comment:
Just call the function directly. _asm is usually not needed, breaks optimizations and just causes a lot of clutter.
```suggestion
    using func_t = (CFxSystemBPSAInterface*)(__thiscall*)(CFxManagerSAInterface* pInterface, const char* szName);
    auto func = reinterpret_cast<func_t>(FUNC_FxManager_c__GetSystemByName);
    return func(m_pInterface, sName);
```